heracles/io.py

```diff
@@ -265,39 +265,48 @@ def _read_complex(hdu):
     return arr
 
 
+def _prepare_result_array(arr, order, size):
+    """Prepare result array for writing."""
+
+    if len(order) == 1:
+        return arr[0]
+    return np.transpose([np.pad(arr[i], (0, size - arr[i].size)) for i in order])
```

Comment:
What does this padding do?
I assume that it moves the axis we are going to use for the fits tables to the back, right?

Reply:
It makes it so that all `ells` etc. have the same length; otherwise, we would have a ragged array.